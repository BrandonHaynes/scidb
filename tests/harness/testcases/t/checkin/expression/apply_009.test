--setup
create array expr1 <a:int64, b:int64>[x=1:5,5,0, y=1:5,5,0]
load(expr1,'${TEST_DATA_DIR}/expr1_1.txt')

--test
apply(expr1, a2, 5 + a * a + 3 * 3 - b + 4 - a * b / a)

--cleanup
remove (expr1)
