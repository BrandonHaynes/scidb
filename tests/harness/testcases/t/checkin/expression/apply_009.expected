Query was executed successfully

{x,y} a,b
{1,1} 2,4
{1,2} 6,8
{1,3} 10,12
{1,4} 14,16
{1,5} 18,20
{2,1} 22,24
{2,2} 26,28
{2,3} 30,32
{2,4} 34,36
{2,5} 38,40
{3,1} 42,44
{3,2} 46,48
{3,3} 50,52
{3,4} 54,56
{3,5} 58,60
{4,1} 62,64
{4,2} 66,68
{4,3} 70,72
{4,4} 74,76
{4,5} 78,80
{5,1} 82,84
{5,2} 86,88
{5,3} 90,92
{5,4} 94,96
{5,5} 98,100

{x,y} a,b,a2
{1,1} 2,4,14
{1,2} 6,8,38
{1,3} 10,12,94
{1,4} 14,16,182
{1,5} 18,20,302
{2,1} 22,24,454
{2,2} 26,28,638
{2,3} 30,32,854
{2,4} 34,36,1102
{2,5} 38,40,1382
{3,1} 42,44,1694
{3,2} 46,48,2038
{3,3} 50,52,2414
{3,4} 54,56,2822
{3,5} 58,60,3262
{4,1} 62,64,3734
{4,2} 66,68,4238
{4,3} 70,72,4774
{4,4} 74,76,5342
{4,5} 78,80,5942
{5,1} 82,84,6574
{5,2} 86,88,7238
{5,3} 90,92,7934
{5,4} 94,96,8662
{5,5} 98,100,9422

Query was executed successfully

